src/bloggy_frontend/src/pages/Posts.jsx

```diff
@@ -38,30 +25,15 @@ const Posts = () => {
       </div>
       {isLoading ? (
         <div className="text-center mt-5">
-          <Spinner animation="border" role="status">
-            <span className="sr-only">Loading...</span>
-          </Spinner>
+          <Spinner animation="border" role="status"></Spinner>
         </div>
       ) : error ? (
         <div className="text-danger text-center mt-5">{error.message}</div>
       ) : (
         <Row xs={1} md={2} lg={3} className="g-4">
           {posts.map((post, index) => (
             <Col key={index}>
```

Comment:
use post id instead of index for key https://react.dev/learn/rendering-lists#why-does-react-need-keys